backend/database_handler/validators_registry.py

```
@@ -1,6 +1,6 @@
 # consensus/domain/state.py
 
-from typing import List
+from typing import List, Callable, Coroutine
```

Ruff (0.11.9)

`typing.Callable` imported but unused

Remove unused import

(F401)